ab-testing/config/lib/fastly/client.test.ts

```diff
@@ -0,0 +1,274 @@
+import { deepEqual, equal, match, rejects } from "node:assert";
```

Comment:
nice set of tests. looks like the only test we may be missing is an assertion for fetching from the correct fastly endpoint?